### ISSUE TITLE
Failing tests in jammy + humble

### ISSUE THREAD

**Issue by orensbruli** [open-rmf/rmf_internal_msgs#47]
## Bug report

Failing tests in jammy + humble

### Required information:

- Operating system and version:
  - Ubuntu jammy
- OpenRMF installation type:
  - source
- OpenRMF version or commit hash
  - 
- ROS distribution and version:
  - Humble
- ROS installation type:
  - Docker container

### Steps to reproduce the bug

```
git clone https://github.com/open-rmf/rmf_internal_msgs
source /opt/ros/humble/setup.sh
colcon build
colcon test --event-handlers=console_cohesion+ --return-code-on-test-failure --packages-select rmf_task_msgs
```

### Expected behavior

Test passed.

### Actual behavior

Failing tests.

**Comment by orensbruli** [comment]
Fixed with https://github.com/open-rmf/rmf_internal_msgs/commit/e70fe9738784cdc8b7bb6897fdbdb64e6a0c4baf
I don't really know why it was not previously failing. I guess it's something introduced with a new version of uncrustify?

**Comment by luca-della-vedova** [comment]
FIxed with #49
